pages/apply.html

```diff
@@ -20,6 +20,34 @@ <h1 class="site-c-section-heading__heading site-c-section-heading__heading--h1">
       <div class="site-c-section-heading__text site-c-section-heading__text--big usa-prose">
         {{ content.page_title.heading_text | replace: 'site.baseurl', site.baseurl | markdownify}}
       </div>
+      <div class="usa-prose">
+        <p>USDS operates on a tour-of-service model, with most people serving for one or two years, and often options to extend up to four years. We'll consider some shorter tours, and time commitments are not binding.
+        </p>
+        <p>
+          <strong>Requirements:</strong>
+          You must be a citizen of the United States to work at USDS. On top of that, you'll need to pass a background check and a drug test. During the COVID-19 pandemic, most people at USDS are working under a distributed model, from many locations across the country. During normal times, most of the teams we work with are in the D.C. metro area, so we ask that you make D.C. or the surrounding area your base during the work week.
+        </p>
+        <p>
+          <strong>Compensation and benefits:</strong>
+Salaries for digital service experts at USDS vary, but don't exceed $170,800, determined by your experience and qualifications. We provide health care and other benefits, such as retirement savings account eligibility. For a wealth of information and different benefit plans offered look on the <a href="https://www.opm.gov/healthcare-insurance/healthcare/plan-information/plans/">OPM website</a>.
```

Comment:
```suggestion
Salaries at USDS vary, but don't exceed $170,800, determined by your experience and skills. We provide health care and other benefits, such as retirement savings accounts. For more information, see the <a href="https://www.opm.gov/healthcare-insurance/healthcare/plan-information/plans/">OPM website</a>.
```

Reply:
Whoops accidentally left this out of the previous commits, adding in the next one!